sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -1741,7 +1741,8 @@ SIREPO.app.directive('appHeaderBrand', function() {
             '<div class="navbar-header">',
               '<a class="navbar-brand" href="/en/landing.html"><img style="width: 40px; margin-top: -10px;" src="/static/img/sirepo.gif" alt="RadiaSoft"></a>',
               '<div class="navbar-brand">',
-                '<a data-ng-href="{{ appUrl || nav.sectionURL(\'simulations\') }}">',
+                '<span data-ng-if="! hasAppLink()">', appInfo.longName, '</span>',
+                '<a data-ng-if="hasAppLink()" data-ng-href="{{ appUrl || nav.sectionURL(\'simulations\') }}">',
                   '<span class="hidden-md hidden-sm">',
```

Comment:
`appUrl ||` is unnecessary. Seems like it would be good to leave the span class the same for both cases?